src/cargo_command.rs

```diff
@@ -61,7 +61,7 @@ impl<'a> CargoCmd<'a> {
 
     /// Run the cargo command and get the output back as a vector
     pub(crate) fn run(&self) -> Result<CargoBuildOutput, Error> {
-        debug!(
+        println!(
```

Comment:
Not sure we want to print this by default

Reply:
Oh, that might have been a debugging artifact. I'll clean that up.

Reply:
See the above output. Without this line, we end up with very little information about what command is being run that executes immediately after that.

Reply:
You could use `info` instead of debug here.